torch/_torch_docs.py

```diff
@@ -7754,3 +7754,126 @@ def merge_dicts(*dicts):
+                            (one pass the last index of the *innermost* dimension). In other words, if False, 
+                            getting lower bound index for each value in :attr:`values` on the corresponding 
+                            *innermost* dimension of the :attr:`sorted_sequence`. If True, getting the upper 
+                            bound index instead. Default value is False.
+    out (Tensor, optional): the output tensor, must be the same size as :attr:`values` if provided.
+
+.. note:: If your use case is always 1-D sorted sequence, :func:`torch.bucketize` is preferred, 
+          because it has less dimension check, slightly better performance.
+
+
+Example::
+
+    >>> sorted_sequence = torch.tensor([[1, 3, 5, 7, 9], [2, 4, 6, 8, 10]]) 
+    >>> sorted_sequence
+    tensor([[ 1,  3,  5,  7,  9],
+            [ 2,  4,  6,  8, 10]])
+    >>> values = torch.tensor([[3, 6, 9], [3, 6, 9]])
+    >>> values
+    tensor([[3, 6, 9],
+            [3, 6, 9]])
+    >>> torch.searchsorted(sorted_sequence, values)
+    tensor([[1, 3, 4],
+            [1, 2, 4]])
+    >>> torch.searchsorted(sorted_sequence, values, right=True)
+    tensor([[2, 3, 5],
+            [1, 3, 4]])
+
+    >>> sorted_sequence_1d = torch.tensor([1, 3, 5, 7, 9])
+    >>> sorted_sequence_1d
+    tensor([1, 3, 5, 7, 9])
+    >>> torch.searchsorted(sorted_sequence_1d, values)
+    tensor([[1, 3, 4],
+            [1, 3, 4]])
+""")
+
+add_docstr(torch.bucketize,
+           r"""
+bucketize(input, boundaries, out_int32=False, right=False, out=None) -> Tensor
+
+Find the indices from the sorted 1-D tensor :attr:`boundaries` such that, if the corresponding values 
+in :attr:`input` were inserted before the indices, the order of :attr:`boundaries` would be preserved. 
+Return a new tensor, same size as :attr:`input`. The returned index satisfies the following rules:
+
+.. list-table:: 
+   :widths: 15 85 
+   :header-rows: 1
+
+   * - :attr:`right`
+     - *returned index satisfies*
+   * - False
+     - ``boundaries[i-1] <= input[m][n]...[l][x] < boundaries[i]``
+   * - True
+     - ``boundaries[i-1] < input[m][n]...[l][x] <= boundaries[i]``
+
+Args:
+    input (Tensor or Scalar): N-D tensor or a Scalar containing the search value(s).
+    boundaries (Tensor): 1-D tensor, must contain a monotonically increasing sequence.
+    out_int32 (bool, optional): indicate the output data type. torch.int32 if True, torch.int64 otherwise. 
+                                Default value is False, ie. default output data type is torch.int64.
+    right (bool, optional): if False, return the first suitable location that is found. If True, return the 
+                            last such index. If no suitable index found, return 0 for non-numerical value 
+                            (eg. nan, inf) or the size of :attr:`boundaries` (one pass the last index). 
+                            In other words, if False, getting lower bound index 
```

Comment:
nit: "if False, gets the lower bound index" (instead of "getting"). Ditto for the next sentence